unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ChatWidgetHUD/ChatHUDView.cs

```diff
@@ -136,11 +139,15 @@ public override void Update()
         if (updateLayoutDelayedFrames > 0)
         {
             updateLayoutDelayedFrames--;
-            
+
             if (updateLayoutDelayedFrames <= 0)
+            {
+                chatEntriesContainer.ForceUpdateLayout(delayed: false);
                 chatEntriesContainer.ForceUpdateLayout(delayed: false);
```

Comment:
Had to add one call to `ForceUpdateLayout`, since chatEntriesContainer was not correctly resized with only one call